### ISSUE TITLE
Key exception got when load_dump Posts and Comments

### ISSUE THREAD

**Issue by rmrf** [HonzaKral/es-django-example#6]
Seems there are exception data inside Posts.xml (item don't have OwnerUserId ) and Comments.xml (item don't have UserId ) file. 

I add bellow code to bypass the key exception error:
add to parse_posts()'s try 

```
_ = data['OwnerUserId']
```

add to parse_comments()'s for loop. 

```
try:
    _ = comment['UserId']
except KeyError:
    continue
```

**Comment by jpabbuehl** [comment]
thx for the tips

**Comment by honzakral** [comment]
This has been fixed by 074b9b2, thanks @varunpurohit76 !
